qemu/tests/multi_nics_stress.py

```diff
@@ -26,16 +26,20 @@ def launch_netperf_client(test, server_ips, netperf_clients, test_option,
             if utils_misc.wait_for(n_client.is_netperf_running, 30, 0, 1,
                                    "Wait netperf test start"):
                 logging.info("Netperf test start successfully.")
-            else:
-                test.error("Can not start netperf client.")
 
     for n_client in netperf_clients:
         if n_client.is_netperf_running():
             left_time = stop_time - time.time()
-            utils_misc.wait_for(lambda: not
-                                n_client.is_netperf_running(),
-                                left_time, 0, 5,
-                                "Wait netperf test finish %ss" % left_time)
+            if utils_misc.wait_for(lambda: not
```

Comment:
Actually, it's hard to say this is a good idea, another way is to modify `is_netperf_running` so that it can capture the correct output.

Reply:
Regarding function is_netperf_running() , I did not find a good solution. Could you  give me some help?

Reply:
Hello @PaulYuuu  Could you give a more detailed explanation? I tried to change the usage of "is_netperf_running()"  to find the PID of netperf running. But this does not seem easy. It calls another method "is_target_running(self, target)". What puzzles me is that I don't understand how the "target"  transfers parameters? Thanks in advance.

Reply:
`target` can be `netperf` and `netserver`. For example, if the netserver is running, the output of it like this:
```
# ps -C netserver
    PID TTY          TIME CMD
 160915 ?        00:00:00 netserver
```
So we need to update the pattern to make it can capture ` 160915 ?        00:00:00 netserver`(`r'^\d+.+%s$' % target`) correctly, but not `re.compile(r"%s" % target, re.I | re.M)`